doc/source/ray-core/scheduling/accelerators.rst

```diff
@@ -622,9 +622,9 @@ This also lets the multi-node-type autoscaler know that there is demand for that
 
     ray.shutdown()
     import ray.util.accelerators
-    import ray._private.ray_constants as ray_constants
+    from ray._common.utils import RESOURCE_CONSTRAINT_PREFIX
 
-    v100_resource_name = f"{ray_constants.RESOURCE_CONSTRAINT_PREFIX}{ray.util.accelerators.NVIDIA_TESLA_V100}"
+    v100_resource_name = f"{RESOURCE_CONSTRAINT_PREFIX}{ray.util.accelerators.NVIDIA_TESLA_V100}"
```

Comment:
let's change this to not use the constant since it isn't a public API

(this will be removed soon in favor of labels)